adaptor/index.html

```diff
@@ -183,7 +183,9 @@
         </tr>
       </tbody>
     </table>
-<br>
+    <p>
+      See the <a href="docs/KnownIssues.html">Known Issues</a> for information about an installer error on Windows Server 2012 R2 with Java 8.
```

Comment:
Suggestion:

**Note:** Using the exact combination of Windows Server 2012 R2 and Java 8 requires <a href=docs/KnownIssues.html>extra step</a> to run installer.